src/containers/color-indicator.jsx

```diff
@@ -30,11 +31,10 @@ const makeColorIndicator = (label, isStroke) => {
             // Flag to track whether an svg-update-worthy change has been made
             this._hasChanged = false;
         }
-        componentWillReceiveProps (newProps) {
-            const {colorModalVisible, onUpdateImage} = this.props;
-            if (colorModalVisible && !newProps.colorModalVisible) {
+        componentDidUpdate (prevProps) {
+            if (prevProps.colorModalVisible && !this.props.colorModalVisible) {
                 // Submit the new SVG, which also stores a single undo/redo action.
-                if (this._hasChanged) onUpdateImage();
+                if (this._hasChanged) this.props.onUpdateImage();
```

Comment:
~I'm not sure whether or not this is important, but if I'm reading this correctly the old code would call `onUpdateImage` from the previous props, where the new code calls it on the new props.~

I looked into where `onUpdateImage` comes from and I'm pretty convinced that (a) it never changes, and (b) if it ever did change, using the new value is probably more correct. So I think this change is fine :)